pkg/operator/controller/status.go

```diff
@@ -148,6 +151,9 @@ func computeOperatorStatusConditions(conditions []configv1.ClusterOperatorStatus
 			"%d ingress controllers available, want %d",
 			ingressesAvailable, numIngresses)
 	}
+	if !versionsEqual {
+		progressingCondition.Message = "configured and current release versions do not match"
```

Comment:
In the case of two successive upgrades, if nothing else updates the `Progressing` condition in between upgrades, it will still have the message from the first upgrade when the second upgrade happens.  We should make sure the message changes between upgrades (simplest way would be to include the version string in the condition's message).

Reply:
Good point. I'll pass the configured version into `computeOperatorStatusConditions()` and include it in the message string.